resources/views/admin/adminUpdate.blade.php

```diff
@@ -46,9 +46,9 @@
                                 @foreach($allUsers as $user)
                                     <tr class="userListItem" id="{{$user->id}}" 
                                         title="Company: {{$user->company}}
-Job Title: {{$user->jobTitle}}
-City: {{$user->city}}
-State: {{$user->state}}">
+                                        Job Title: {{$user->jobTitle}}
+                                        City: {{$user->city}}
+                                        State: {{$user->state}}">
                                         <th scope="row">{{$user->id}}</th>
```

Comment:
Alex initially did this to have the user info popup on hover... (which is why it needed to be not indented).

Could you change this to be a proper popover/tooltip? See:
https://getbootstrap.com/docs/4.4/components/tooltips/